Plugin Cubot/src/main/java/net/simon987/cubotplugin/event/RealLifeTimer.java

```diff
@@ -0,0 +1,39 @@
+package net.simon987.cubotplugin.event;
+
+import java.util.Timer;
+import java.util.TimerTask;
+
+public class RealLifeTimer{

```

Comment:
I'm not sure I understand what you're trying to do here
Please add this feature in a separate PR

Reply:
I removed this file